src/content/en/updates/2019/08/twas-quickstart.md

```diff
@@ -0,0 +1,221 @@
+
+## Clone and customize the example repo {: #clone-and-customize-the-example-repo }
+
+The [svgomg-twa](https://github.com/GoogleChromeLabs/svgomg-twa) repo contains an example TWA that
+you can customize to launch your website:
+
+Clone the project (`git clone https://github.com/GoogleChromeLabs/svgomg-twa.git`).
+Import the Project into Android Studio, using File > New > Import Project, and select the folder to
+which the project was cloned.
+
+Open the app’s `build.gradle` (there are two, you want the module one at app/build.gradle, not the
+top level one) and modify the values in `twaManifest`:
+
+* Change `hostName` to point to your website.
+* Your website must be available on HTTPS, though you omit that from the hostName field.
+  Change `name` to whatever you want.
+* Change `applicationId` to something specific to your project.
+  This translates into the app’s package name and is how the app is identified on the Play Store -
+  no two apps can share the `applicationId` and if you change it you’ll need to create a new Play
+  Store listing.
+
+## Build and run {: #build-and-run }
+
+In Android Studio hit Run, Run ‘app’ (replace ‘app’ with your module name if you’ve changed it) and
+the TWA will be built and run on your device!
+You’ll notice that the top bar is still visible, this is because we haven’t set up our Digital Asset
+Links yet, but first...
+
+### A note on signing keys {: #a-note-on-signing-keys }
+
+**Digital Asset Links** take into account the key that an APK has been signed with and a common
+cause for verification failing is to have the signature wrong.
+When you hit *Run* or *Build APK* in Android Studio, the APK will be created with your developer
+**debug key**, which Android Studio automatically generated for you.
+
+If you deploy your app to the Play Store you’ll hit *Build > Generate Signed APK*, which will use a
+different signature, one that you’ll have created yourself (and protected with a password).
+That means that if your **Digital Asset Links** file specifies your **production key**, verification
+will fail when you build with your **debug key**.
+This also can happen the other way around - if the Digital Asset Links file has your **debug key**
+your TWA will work fine locally, then when you sign it and upload it to Play it will fail to verify.
+
+You can put both your debug and production key in your asset link file (see Adding More Keys below),
+but your debug key is less secure (anyone who gets a copy of the file can use it).
+Finally, if you have your app installed on your device with one key, you can’t install the version
+with the other key - you must uninstall the previous version first.
+
+### Building with debug keys {: #building-with-debug-keys }
+
+Just hit *Run ‘app’* (replace ‘app’ with your module name if you’ve changed it).
+
+### Building with release keys {: #building-with-release-keys }
+
+Hit *Build > Generate Signed APK*, choose APK and on the next page hit “Create New” to create a new
```

Comment:
Should this be an ordered list?

Reply:
Hey, I think I addressed this in the previous upload (although not marked this as resolved). Marking it as resolved now.